README.md

```diff
@@ -218,6 +218,147 @@ You should create and keep your models under this folder. Note that we have appl
 
 Once your model is ready to submit, simply create a pull request on the github repo and let us know in our [Discord](https://discord.com/invite/BJbYSRDdtr).
 
+
+# SDK Command Documentation 
+## Help command
+
+All the commands accept `-h` parameter for help, e.g.:
+
+```
+>credmark-dev -h
+
+usage: credmark-dev [-h] [--log_level LOG_LEVEL] [--model_path MODEL_PATH] [--manifest_file MANIFEST_FILE]
```

Comment:
This is an older version that doesn't have the new commands like "describe" and "models"